examples/CairoUI/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+# CMake file for DISTRHO Plugins #
+# ------------------------------ #
+
+dpf_add_plugin(d_cairoui
+  TARGETS jack ladspa dssi lv2 vst
+  UI_TYPE cairo
+  FILES_DSP
+      CairoExamplePlugin.cpp
+  FILES_UI
+      DemoWidgetBanner.cpp
+      DemoWidgetClickable.cpp
+      CairoExampleUI.cpp)
+
+target_include_directories(
```

Comment:
do we require this line to be in every single cmake file?
is there a reason it cant or shouldnt be done inside the `dpf_add_plugin` function?

Reply:
This function is able to work in arbitrary source hierarchy, as opposed to the fixed `plugins/<name>` directory. I didn't have really a reason to enforce it so I let it be that way.
Otoh, there wasn't a concept like a "plugin home" directory (and consequently not a default path for "plugin info" includes).
Depending on preference, I can revise the `dpf_add_plugin` interface for such concept.

Reply:
you mean the dir where the cmake file resides (and thus where the dpf function is called from) might be different from the plugin code dir?

Reply:
> might be different from the plugin code dir?

Yes that is supposed to be possible, as currently written. (but not checked)

Reply:
Ok that makes sense.
Seems like good practice, and we want to recommend those.
Let those lines stay then :)